### PR TITLE
Flatpak Appdata update

### PR DESCRIPTION
**Pull request by Laserlicht**
Add new links / adjust Links to "Github Pages" if possible.

By the way: The new Flathub Beta looks really nice. https://beta.flathub.org/de/apps/io.github.ihhub.Fheroes2

### CODE REVIEW

File: script/packaging/common/fheroes2.appdata.xml [hunk 1]
```diff
@@ -21,8 +21,12 @@
   <url type="homepage">https://ihhub.github.io/fheroes2/</url>
   <url type="bugtracker">https://github.com/ihhub/fheroes2/issues</url>
   <url type="faq">https://github.com/ihhub/fheroes2/wiki/F.A.Q.</url>
-  <url type="translate">https://github.com/ihhub/fheroes2/blob/master/docs/TRANSLATION.md</url>
+  <url type="help">https://ihhub.github.io/fheroes2/INSTALL.html</url>
+  <url type="translate">https://ihhub.github.io/fheroes2/TRANSLATION.html</url>
   <url type="donation">https://www.patreon.com/fheroes2</url>
+  <url type="contact">https://discord.gg/xF85vbZ</url>
```
Comment (ihhub) [inline script/packaging/common/fheroes2.appdata.xml:27]: Does contact need to be a URL? Can we put an email?

File: script/packaging/common/fheroes2.appdata.xml [hunk 2]
```diff
@@ -21,8 +21,12 @@
...
+  <url type="vcs-browser">https://github.com/ihhub/fheroes2/</url>
```
Comment (ihhub) [inline script/packaging/common/fheroes2.appdata.xml:28]: Is a slash at the end of the URL needed?

**Review by ihhub — COMMENTED** [review]
Hi @Laserlicht , I left 2 comments. Could you please take a look?

File: script/packaging/common/fheroes2.appdata.xml (hunk 1 above)
Comment (Laserlicht) [reply script/packaging/common/fheroes2.appdata.xml:27]: Maybe it is possible, but we would be the first: https://github.com/search?p=1&q=org%3Aflathub+%22url+type%3D%5C%22contact%22&type=Code

File: script/packaging/common/fheroes2.appdata.xml (hunk 2 above)
Comment (Laserlicht) [reply script/packaging/common/fheroes2.appdata.xml]: No slash needed

**Comment by ihhub** [comment]
@Laserlicht , thanks a lot for this update!

**Pull request merged**